tests/plugins/font/manual/command/removestyle.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.14.0, feature, 3728
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, font, enterkey, elementspath, notification, sourcearea
+
+General Note:
+* Font and font size now use commands instead of direct operation on editor's content.
+* When font or fontSize command is executed then notification should be displayed in the editor and browser's console.
+* Please repeat steps for both font and font size features
+
+1. Select styled text and remove font/fontSize from it.
+2. Try to remove style from not styled text.
```

Comment:
Also here, bullet list and numbered list are mixed which results in weird-looking bullet list.